actions/graybox/promote-worker.js

```diff
@@ -61,26 +61,27 @@ async function main(params) {
     
     // process data in batches
     const previewStatuses = [];
-
+    let failedPreviews = '';
     if (helixUtils.canBulkPreview()) {
-        const paths = []; 
-        
+        const paths = [];
         batchArray.forEach((batch) => {
             batch.forEach((gbFile) => paths.push(handleExtension(gbFile.filePath)));
         });
         
-        previewStatuses.push(await helixUtils.bulkPreview(paths, helixUtils.getOperations().PREVIEW, experienceName));
+        const previewStatuses = await helixUtils.bulkPreview(paths, helixUtils.getOperations().PREVIEW, experienceName);
 
         logger.info(`Preview Statuses >> ${JSON.stringify(previewStatuses)}`);
+        failedPreviews = previewStatuses.filter((status) => !status.success).map((status) => status.path);
+        const promoteErrors = failedPreviews.length > 0;
 
-        const failedPreviews = previewStatuses.filter((status) => !status.success).map((status) => status.path);
+        logger.info(`Failed Previews: ${JSON.stringify(failedPreviews)}`);
     }
 
     // Update project excel file with status (sample)
     logger.info('Updating project excel file with status');
     const curreDateTime = new Date();
     const { projectExcelPath } = appConfig.getPayload();
-    const excelValues = [['Sample Excel Update', toUTCStr(curreDateTime), 'sukamat@adobe.com', '']];
+    const excelValues = [['Sample Excel Update', toUTCStr(curreDateTime), 'sukamat@adobe.com', 'Failed Previews: \n' + failedPreviews.join('\n')]];
```

Comment:
Feel free to update the 3rd column as you see fit. The email for just an example added to get started.